build-system/tasks/visual-diff.rb

```diff
@@ -222,31 +220,55 @@ def load_visual_tests_config_json
 end
 
 
+# Configures the Chrome browser (optionally in headless mode)
+def configure_browser
+  if ARGV.include? '--headless'
+    chrome_args = %w(no-sandbox disable-extensions headless disable-gpu)
```

Comment:
Why `--disable-gpu` in `--headless`?

Reply:
It's recommended by Chrome, since it won't work in headless mode without that flag on some platforms. See documentation link in the PR description.